oca_port/port_addon_pr.py

```diff
@@ -945,7 +945,7 @@ def _get_original_pr(self, commit: g.Commit, fallback_pr=None):
             raw_data = self.app.github.get_original_pr(
                 self.app.upstream_org,
                 src_repo_name,
-                self.app.from_branch.name,
+                self.app.source_version,
```

Comment:
The issue with this patch is it makes incompatible the original PR detection on specific repositories not following Odoo version scheme for branches (like a project repository with a branch named `master`/`main`).

We could request GH for both branches otherwise...